release_dashboard/lib/widgets/common/dialog_prompt.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Widget that prompts an alert dialogue to the current window and forces the user to choose between two options.
+Future<String?> dialogPrompt(
```

Comment:
Where is `conductor.clean()` called? It's not uploaded

Reply:
Oh yeah, I am justing giving it as an example. This PR does not connect with `conductor clean`. It just makes the UI. I will make another PR to connect it with `conductor clean`.

Reply:
It's misleading then. You can throw an UnimplementedException then to indicate it's WIP, but I would expect it to only be a few extra lines to connect it (granted it may require the state PR to land before this one)

Reply:
drive by comment: I don't think you need to hook this up with the core code--you know where the file is, just delete it.